yogstation/code/modules/guardian/guardianstats.dm

```diff
@@ -75,3 +75,33 @@
 		var/datum/guardian_ability/minor/minor_ability = A
 		stats_info += minor_ability.name
 	return stats_info.Join(", ")
+
+/datum/guardian_stats/proc/weaken(times = 1)//reduces a random stat x times
+	for(var/i = 0; i < times; i++)
+		var/list/reduction = list("damage", "defense", "speed", "potential", "range")
+		shuffle(reduction)
+		var/found = FALSE
+		for(var/option in reduction)
```

Comment:
suggest some failsafe in case all the stats end up at 1 and this loops

Reply:
It should stop when it goes through all options, shouldnt it?

Reply:
i mifhr vw crazy